core/domain/exp_jobs_one_off_test.py

```diff
@@ -668,11 +978,390 @@ def test_migration_job_skips_deleted_explorations(self):
         with self.assertRaisesRegexp(Exception, 'Entity .* not found'):
             exp_services.get_exploration_by_id(self.NEW_EXP_ID)
 
+    def test_exploration_migration_job_produces_no_output(self):
+        """Test that Exploration Migration job produces no output."""
+        exp_domain.Exploration.create_default_exploration(
+            self.VALID_EXP_ID, title='title', category='category')
 
-class ExplorationStateIdMappingJobTest(test_utils.GenericTestBase):
-    """Tests for the ExplorationStateIdMapping one off job."""
+        # Start migration job on sample exploration.
+        job_id = exp_jobs_one_off.ExplorationMigrationJobManager.create_new()
+        exp_jobs_one_off.ExplorationMigrationJobManager.enqueue(job_id)
+        self.process_and_flush_pending_tasks()
 
-    EXP_ID = 'eid'
+        self.assertEqual(
+            exp_jobs_one_off.ExplorationMigrationJobManager.get_output(job_id),
+            []
+        )
+
+
+class InteractionAuditOneOffJobTests(test_utils.GenericTestBase):
```

Comment:
Does this job work for multiple explorations? And return all the exp_id, state_name pairs? Can we add a test for that?

Reply:
Done.